### PR TITLE
Additional code coverage

### PR DESCRIPTION
**Pull request by jrabbott**
Adds additional unit tests to cover the following methods in abstractions:

EducationBenchmarking.Platform.Domain

- Class UnknownProximitySort method Sort
- Class SenProximitySortmethod method Sort
- Class SimpleProximitySort method Sort
- Class BestInClassProximitySort method Sort

EducationBenchmarking.Platform.Functions

- Class JsonContentResult constructor
- Class HttpRequestExtensions method GetCorrelationId
- Class HttpRequestExtensions method ReadAsJson
- Class QueryCollectionExtensions method GetPagingValues

### CODE REVIEW

File: platform/EducationBenchmarking.Platform.sln [hunk 1]
```diff
@@ -65,9 +65,13 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "data", "data", "{1D110B5A-0
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EducationBenchmarking.Platform.Search", "src\search\EducationBenchmarking.Platform.Search\EducationBenchmarking.Platform.Search.csproj", "{B6991DB1-7945-4348-86ED-B00A3AC8B889}"
 EndProject
+<<<<<<< HEAD
```
Comment (Faizi52996) [inline platform/EducationBenchmarking.Platform.sln:68]: is it is left after resolving conflicts?

File: platform/EducationBenchmarking.Platform.sln [hunk 2]
```diff
@@ -65,9 +65,13 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "data", "data", "{1D110B5A-0
...
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EducationBenchmarking.Platform.Functions", "src\abstractions\EducationBenchmarking.Platform.Functions\EducationBenchmarking.Platform.Functions.csproj", "{AD3DC4F8-B866-40FE-A456-B18BA3158403}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EducationBenchmarking.Platform.Domain", "src\abstractions\EducationBenchmarking.Platform.Domain\EducationBenchmarking.Platform.Domain.csproj", "{3143ABD3-CB9E-4A8E-A501-6D2B151E6D1C}"
+=======
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "EducationBenchmarking.Platform.Shared.Tests", "tests\EducationBenchmarking.Platform.Shared.Tests\EducationBenchmarking.Platform.Shared.Tests.csproj", "{55E94E05-2406-4B20-9C5B-F3BF9D378104}"
+>>>>>>> c15eaa5 (Add init shared test project)
```
Comment (Faizi52996) [inline platform/EducationBenchmarking.Platform.sln:74]: while resolving conflicts few additional lines have been left.

File: platform/tests/EducationBenchmarking.Platform.Shared.Tests/WhenGetPagingValuesIsCalled.cs [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            {
+                { "page", "invalid" },
+                { "pageSize", "invalid" }
+            });
+
+        var result = query.GetPagingValues();
+
+        Assert.Equal(1, result.Page);
+        Assert.Equal(10, result.PageSize);
+    }
```
Comment (Faizi52996) [inline platform/tests/EducationBenchmarking.Platform.Shared.Tests/WhenGetPagingValuesIsCalled.cs:50]: is it expected to return page and size when invalid values are passed?

File: platform/EducationBenchmarking.Platform.sln (hunk 1 above)
Comment (jrabbott) [reply platform/EducationBenchmarking.Platform.sln]: Good spot. Pushed a fix

File: platform/EducationBenchmarking.Platform.sln (hunk 2 above)
Comment (jrabbott) [reply platform/EducationBenchmarking.Platform.sln]: Good spot. Pushed a fix

File: platform/tests/EducationBenchmarking.Platform.Shared.Tests/WhenGetPagingValuesIsCalled.cs (hunk 3 above)
Comment (jrabbott) [reply platform/tests/EducationBenchmarking.Platform.Shared.Tests/WhenGetPagingValuesIsCalled.cs:50]: What is the logic in the code that is under test?

**Review by Faizi52996 — APPROVED** [review]

**Pull request merged**